style.scss

```diff
@@ -12,6 +11,12 @@ body {
     font-family: Arial, Helvetica, sans-serif;
 }
 
+a {
+    text-decoration: none;
+    border: none;
+   margin: 5px;
```

Comment:
Yep, that's right. But you applied `margin: 5px` for every single link on the site. Removing text-decoration is ok if you know that you won't use the default value for links. But setting any kind of offset globally for all links it's not a good idea. So, please create a class for the social media links, like `.social-link`, and set this margin there.